src/Models/IoTWorkbenchProjectBase.ts

```diff
@@ -39,25 +45,29 @@ export abstract class IoTWorkbenchProjectBase {
     }
     const iotWorkbenchProjectFile =
         path.join(projectFileRootPath, FileNames.iotworkbenchprojectFileName);
-    const devcontainerFolderPath =
-        path.join(projectFileRootPath, FileNames.devcontainerFolderName);
     if (!await FileUtility.fileExists(scaffoldType, iotWorkbenchProjectFile)) {
       return ProjectHostType.Unknown;
-    } else if (await FileUtility.directoryExists(
-                   scaffoldType, devcontainerFolderPath)) {
+    }
+    const iotworkbenchprojectFileString =
+        await FileUtility.readFile(
+            scaffoldType, iotWorkbenchProjectFile, 'utf8') as string;
+    if (iotworkbenchprojectFileString !== '') {

```

Comment:
Since null, undefined and "" are falsy value,  you can just boolean evaluation, such as
if (iotworkbenchprojectFileString !== '')  => if (iotworkbenchprojectFileString)

Reply:
Fixed.